CRM/MembersOnlyEvent/Hook/PageRun/Register.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+use CRM_MembersOnlyEvent_Hook_PageRun_Base as Base;
```

Comment:
I would change this to : 

use CRM_MembersOnlyEvent_Hook_PageRun_Base as PageRunBase;

same applies to the other hook class.